### PR TITLE
Transport Memory and Metrics

### PR DESCRIPTION
**Pull request by ozsay**

### CODE REVIEW

File: packages/metrics/src/index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import forEach from 'lodash/forEach';
+
+let metrics = {};
+const ONE_MINUTE = 60 * 1000;
+const PUBLISH_INTERVAL = process.env.METRICS_PUBLISH_INTERVAL || ONE_MINUTE;
```
Comment (sloops77) [inline packages/metrics/src/index.js]: STELLAR_METRICS_PUBLISH_INTERVAL

File: packages/transport-memory/test/transport-memory.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Created by arolave on 14/05/2017.
+ */
+xdescribe('no tests', () => {
+  xit('no test', () => {
+
```
Comment (sloops77) [inline packages/transport-memory/test/transport-memory.test.js]: i think this needs tests! 90% coverage at least

File: packages/metrics/test/metrics.test.js [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        });
+  });
+
+  xit('gets metrics from the subscription', (done) => {
+    // TODO needs to figure out timers faking
+    const stellar = createStellar();
+    stellar.handler.get(resource1, () => {});
+
+    jest.useFakeTimers();
+    runMetrics({ handler: stellar.handler, pubSub: stellar.pubSub }, service, '.*');
```
Comment (sloops77) [inline packages/metrics/test/metrics.test.js]: hack your interval value somehow and then this should work fine without fakeTimers

File: packages/transport-memory/test/transport-memory.test.js (hunk 2 above)
Comment (ozsay) [inline packages/transport-memory/test/transport-memory.test.js]: yeah, I told you about it. for now we use it only to test the metrics.. if we will expand our usage, we will add tests. I can't see a point to do it now

Comment (sloops77) [inline packages/transport-memory/test/transport-memory.test.js]: ok

File: packages/metrics/package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "babel-jest": "^20.0.3",
+    "bluebird": "3.4.6",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb-base": "^11.2.0",
+    "eslint-plugin-import": "^2.3.0",
+    "jest": "^20.0.4",
+    "rimraf": "^2.6.1"
+  },
+  "dependencies": {
+    "lodash": "4.15.0"
```
Comment (sloops77) [inline packages/metrics/package.json]: correct version of lodash: ^4.17.4

**Pull request merged**